src/app/content/highlights/components/EditCard.tsx

```diff
@@ -75,6 +76,19 @@ const EditCard = React.forwardRef<HTMLElement, EditCardProps>((props, ref) => {
 
   useOnEsc(element, props.isFocused, cancelEditing);
 
+  // Overwrite .focus method on element.current to focus textarea when focusing this element.
+  // It is done this way instead of using React.useImperativeHandle because there are other
+  // functions that are being used and not all of them require overrides.
+  React.useEffect(() => {
+    if (element.current) {
+      (element.current as any).focus = () => {
```

Comment:
i don't think we should be monkeypatching the element focus function, this is terrifying to me.